changelogs/v216/node_yarn.md

```diff
@@ -0,0 +1,8 @@
+## Ruby apps now default to Node version 12.16.1 and Yarn version 1.22.0
+
+Applications using the `heroku/ruby` buildpack that do not have a version of Node installed by another buildpack (such as the `heroku/nodejs` buildpack) will now receive:
+
+- Node version 12.16.1
+- Yarn version 1.22.0
```

Comment:
How do you feel about also listing the previous versions, so that customers know what version to downgrade to, should they encounter issues? (Otherwise I guess they would need to find this PR, or inspect old build logs to see what versions were being mentioned in build output?) :-)

Reply:
I like it. I'll add that. Great feedback, this type of comment is exactly why I started putting changlog messages in the PR.

Reply:
Yeah great idea to include them in the repo :-)

This conversation also reminded me of the great changelog entries that eslint uses, where it has an explicit "here's how to get back the old behaviour" code snippet for breaking changes. eg:
https://eslint.org/docs/user-guide/migrating-to-6.0.0#eslint-recommended-changes